moxin-frontend/src/landing/model_card.rs

```diff
@@ -304,3 +387,81 @@ impl Widget for ModelCard {
         self.view.draw_walk(cx, scope, walk)
     }
 }
+
+impl WidgetMatchEvent for ModelCard {
+    fn handle_actions(&mut self, cx: &mut Cx, actions: &Actions, scope: &mut Scope) {
+        let widget_uid = self.widget_uid();
+
+        if self.link_label(id!(view_all_button.link)).clicked(actions) {
+            cx.widget_action(
+                widget_uid,
+                &scope.path,
+                ModalAction::ShowModalView(id!(model_card_view_all_modal_view)[0]),
```

Comment:
I'm pretty sure `id!(model_card_view_all_modal_view)[0]` is equivalent to `live_id!(model_card_view_all_modal_view)`, in case you want to avoid the `[0]` stuff